### PR TITLE
Adds the Bushido and the Clockwork religion

### PR DESCRIPTION
**Pull request by jknpj**
kimono
ratvar
Bushido gives you a sakura kimono and the and clockwork gives you a full clockwork cultist getup and allow you to choose the slab skin for your bible.

:cl:
 * rscadd: Adds the clockwork and bushido religion
 * rscadd: Adds the sakura kimono to the list of premium autodrobe clothing
 * rscadd: Adds the clockwork hood, robe and boots as contraband items in the piety vending

### CODE REVIEW

**Comment by ghost** [comment]
>TG refugee comes
>Immediately valids the clockwork cult chaplain

**Comment by SonixApache** [comment]
clockcreep

**Comment by Kurfursten** [comment]
That kimono sprite is certainly... something

In any case, both costumes should be available at either costume or piety vendor

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1253,37 +1253,6 @@ var/global/num_vending_terminals = 1
...
-		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/blebweiser = 50,
-		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/bluespaceribbon = 40,
-		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/codeone = 50,
-		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/gibness = 50,
-		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/orchardtides = 50,
-		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/sleimiken = 50,
-		/obj/item/weapon/reagent_containers/food/drinks/soda_cans/strongebow = 50,
-		)
-
-	pack = /obj/structure/vendomatpack/offlicence
```
Comment (SonixApache) [inline code/game/machinery/vending.dm]: good job

**Comment by SonixApache** [comment]
you fucked it

**Comment by jknpj** [comment]
shhh you saw nothin

**Pull request merged**